### ISSUE TITLE
Collect Computational Papers (Computational Measurements and Proxies)

### ISSUE THREAD

**Issue by xehu** [Watts-Lab/team-process-map#4]
Next, we need to identify the features in the causal diagram that can be measured for real teams, particularly in a computational setting. We will first specify the kinds of data that will be easy to access (e.g., text-based chats of teams and other observational data) and the kinds of data that will be harder to access, and would therefore be considered “stretch” features (e.g., continuous survey measures, experimental manipulations). This will allow us to prioritize features that will be measurable for our data.

From there, evaluate each of the features that were generated and diagrammed in the earlier deliverables and determine how it should be measured in a team. For example, imagine that one of the variables is “politeness.” We should also do additional literature review on how politeness can be computationally measured (e.g., it turns out that there is a wealth of research on how to measure politeness computationally, such as this one: https://www.cs.cornell.edu/~cristian/Politeness.html).

**To summarize, we should:**

- [ ] Pull elements from the FMM and document them on a Computational Tab
- [ ] If they are qualitative / survey data, identify "computational proxies"
- [ ] Add in purely computational measurements from the literature
